docs/4.3/enterprise/workflow/ssh_approval_mattermost.md

```diff
@@ -0,0 +1,152 @@
+This will allow us to create a new bot account that the Teleport bot will use.
+
+Go back to your team, then Integrations -> Bot Accounts -> Add Bot Account.
+
+The new bot account will need Post All permission. 
+
+The confirmation screen after you've created the bot will give you the access token.
+ We'll use this in the config later.
+
+#### Create an access-plugin role and user within Teleport 
+First off, using an existing Teleport Cluster, we are going to create a new Teleport
+ User and Role to access Teleport.
+
+#### Create User and Role for access. 
+Log into Teleport Authentication Server, this is where you normally run `tctl`. Don't
+ change the username and the role name, it should be `access-plugin` for the plugin 
+ to work correctly.
+
+```bash
+$ cat > rscs.yaml <<EOF
+kind: user
+metadata:
+  name: access-plugin
+spec:
+  roles: ['access-plugin']
+version: v2
+---
+kind: role
+metadata:
+  name: access-plugin
+spec:
+  allow:
+    rules:
+      - resources: ['access_request']
+        verbs: ['list','read','update']
+    # teleport currently refuses to issue certs for a user with 0 logins,
+    # this restriction may be lifted in future versions.
+    logins: ['access-plugin']
+version: v3
+EOF
+
+# ...
+$ tctl create -f rscs.yaml
+```
+
+#### Export access-plugin Certificate
+Teleport Plugin uses the `access-plugin`role and user to peform the approval. We 
+export the identify files, using [`tctl auth sign`](https://gravitational.com/teleport/docs/cli-docs/#tctl-auth-sign).
+
+```bash
+$ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=8760h
+# ...
+```
+
+The above sequence should result in three PEM encoded files being generated: auth.crt,
+ auth.key, and auth.cas (certificate, private key, and CA certs respectively).  We'll
+  reference these later in the bot config, and move them to an appropriate directory.
+
+_Note: by default, tctl auth sign produces certificates with a relatively short lifetime. For production deployments, the --ttl flag can be used to ensure a more practical certificate lifetime. --ttl=8760h exports a 1 year token_
+ 
+
+## Downloading and installing the plugin
+
+The recommended way to run Teleport Mattermost plugin is by downloading the release version and installing it: 
+
+```bash
+$ wget https://get.gravitational.com/teleport-mattermost-v0.0.1-linux-amd64-bin.tar.gz
+$ tar -xzf teleport-mattermost-v0.0.1-linux-amd64-bin.tar.gz
+$ cd teleport-mattermost
+$ ./install
+$ which teleport-mattermost
+/usr/local/bin/teleport-mattermost
+```
+
+### Building from source
+
+```bash
+
+# Checkout teleport-plugins
+git clone git@github.com:gravitational/teleport-plugins.git
+cd teleport-plugins
+
+cd access/mattermost
+make
+```
+
+### Configuring Mattermost bot
+
+Mattermost Bot uses a config file in TOML format. Generate a boilerplate config by 
+running the following command: 
+
+```
+teleport-mattermost configure > /etc/teleport-mattermost.yml
```

Comment:
I think the path is `/etc/teleport-mattermost.toml` if it's a TOML file.

Also you likely won't be able to write to `/etc/teleport` if you're not `root`, so better to write the file to the local directory and then copy it.

```bash
$ teleport-mattermost configure > teleport-mattermost.toml
$ sudo mv teleport-mattermost.toml /etc
```

Reply:
Thanks Gus, I've updated the rest of the examples.